pandasai/core/chart_generator.py

```diff
@@ -0,0 +1,19 @@
+from abc import ABC, abstractmethod
+
+class ChartGenerator(ABC):
+    """
+    Abstract class "ChartGenerator" comprises the set of abstract generate_<chart> methods to be 
+    implemented in the child/sub classes for chart generation library packages
+    """
+ 
+    @abstractmethod
+    def generate_bar(self, dataframe, x, y, **kwargs):
+        pass
+
+    @abstractmethod
+    def generate_line(self, dataframe, x, y, **kwargs):
+        pass
+
+    @abstractmethod
+    def generate_scatter(self, dataframe, x, y, **kwargs):
+        pass
```

Comment:
Pls add an empty line at the end of a module. See [[1](https://unix.stackexchange.com/a/18789)], [[2](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3)].